FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Tan Tan on 5/31/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FeedImageMapper {
+	struct Root: Decodable {
```

Comment:
The `Root` struct is only used inside the `FeedImageMapper`, so it can be private.

```suggestion
	private struct Root: Decodable {
```